packages/core/src/components/Shadows/EditShadowModal.tsx

```diff
@@ -335,10 +362,14 @@ export function EditShadowModal({
                       color: index === shadowIndex ? 'black' : 'gray',
                       width: '65px',
                       margin: '10px',
+                      position: 'relative',
                     }}
                     onClick={() => setShadowIndex(index)}
                   >
                     {index + 1}
+                    <Box position="absolute" top="-5px" right="-5px">
+                      <CloseButton size="sm" onClick={handleDelete(index)} />
```

Comment:
let's remove the hover styling and instead add our own _hover styling to make the icon bolder 

also, what does size xs look like?

Reply:
* default (no hover state) => no background color && normal icon boldness
* on hover =>  no background color && icon gets bolder
Is this the desired behaviour

Reply:
yes! @noobCode-69